### ISSUE TITLE
ESP IDF custom endpoints for BLE

### ISSUE THREAD

**Issue by noorhaq** [espressif/esp-idf-provisioning-ios#53]
In the sdk provided I am unable to find information on how to send data to custom endpoints added in esp idf from mobile app.

Please help.

**Comment by vikas-chandra-mnnit** [comment]
Hi @noorhaq, [this](https://espressif.github.io/esp-idf-provisioning-ios/Classes/ESPDevice.html#/s:12ESPProvision9ESPDeviceC8sendData4path4data17completionHandlerySS_10Foundation0D0VyAJSg_AA15ESPSessionErrorOSgtctF) API can used to send data to custom endpoints.

**Comment by noorhaq** [comment]
I think this is for the softAp provisioning will it work for ble provisioning too.
Because I tried using this function and it didn't work.
The parameters were 
```
sendData('endpoint', 'data', handler);
Also tried
sendconfigdata and sendsession data apis too
```

**Comment by vikas-chandra-mnnit** [comment]
Yes, this works for BLE too, make sure your custom endpoint characteristic is discoverable by SDK.
